### PR TITLE
V1.3

### PR DESCRIPTION
**Pull request by dneanover**
Updates to include Skunk, the role of the Line judge, a glossary of terms, and more.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,37 +4,64 @@
...
+If the paddle of the blocker is hit while attempting to block a shot, it counts as his/her deflection and will count against the blocker if it goes off the table.  If any part of the blocker is over the table and hit while blocking, the ball is live and the responsibility the individual whom was hit.  
 
+##Scoring
 Jollyballs operates on side-out scoring, meaning that only the team that serves can score a point. If the team who received the serve is unable to return the ball to the table in the allotted number of touches, the team who serves receives a point. If the team who served faults, or the other team prevents them from returning the ball, the receiving team now is
 in possession of the ball and rotates positions clockwise.
 
+##Officiating
+Jollyballs is first and foremost a self-officiated sport.  It is an expectation that players recognize when they have crossed the line, hit the ball twice, or broken the rule.  In the event that either side could have been at fault and it is uncertain whom is at fault, a re-do should be declared.
```
Comment (tyleramos) [inline README.md]: Small typo: "or broken a rule."

File: README.md [hunk 2]
```diff
@@ -4,37 +4,64 @@
...
 
+##Officiating
+Jollyballs is first and foremost a self-officiated sport.  It is an expectation that players recognize when they have crossed the line, hit the ball twice, or broken the rule.  In the event that either side could have been at fault and it is uncertain whom is at fault, a re-do should be declared.  
+
+In especially competitive matches, it is not uncommon to have a line judge present.  The line judge has the final say on line infractions and any other contentious plays.  It is important for the line judge to sit along the net so that they have the optimal vantage point to officiate the game.
+
+## End of Game/Match:
+To end the game, the serving team must first state the score then announce "Jolly Point."  Without the declaration of "Jolly Point," the receiving team can declare the point invalid and request a re-do.  On match point, if a team does not declare "Super Jolly Point" or "Jolliest Point" then the same rules apply for "Jolly Point."
+
+At the conclusion of the game, players of both sides line up and shake hands or exchange a words. This is an expected and time-honored tradition.
```
Comment (tyleramos) [inline README.md]: "At the conclusion of the match"

File: README.md [hunk 3]
```diff
@@ -4,37 +4,64 @@
...
+
+## Round Robin Rules:
+* If any multiple of 3 greater than 6 players all would like to play at the same time, a round robin of Jollyballs can be initiated.  In this scenario, the first three players to motion for Jollyballs become the captains following a snake draft selection order.  
+
+* In lieu of traditional matches of three games, teams will rotate sides of the table and opponents after each individual game, rotating from one side of the table, to the other side, then to off the table.  If there is a tie between two teams for being eliminated off the table, the tie breaker will be decided by the total number of points between their two games.
+
 ## Additional Rules:
-* In Jollyballs, all surfaces, with the exception of the floor are live. If the ball hits off the wall, ceiling, tabletop, or any other nearby surface; the ball is playable.
+* In Jollyballs, all surfaces, with the exception of the floor are live. If the ball hits off the wall, ceiling, tabletop, or any other nearby surface; the ball is playable.  If the ball comes to a rest in an accessible place, you have less than three seconds to strike the ball.
```
Comment (tyleramos) [inline README.md]: Didn't know this last detail!

File: README.md [hunk 4]
```diff
@@ -4,37 +4,64 @@
...
 
-* A player may hit the ball off of their body but cannot carry, toss, or throw the ball.
+* A player may hit the ball off of their body but cannot carry, toss, or throw the ball.  Hitting the ball off of your body is known as a Scoop and does not count as a hit.
```
Comment (tyleramos) [inline README.md]: Oh, I question whether the scoop should be counted as a hit.  I can get behind it if it's a block, but could be dangerous not counting it normally as a hit.

File: Glossary.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Jollyballs Glossary
+
+## General Terms
+* *VTS*- VTS stands for Value the Serve.  When a player faults on a serve, it is common to utter "VTS" to him or her.
```
Comment (fatcatt316) [inline Glossary.md]: Might want some quotes around `Value the Serve` here.

File: Glossary.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+* *Trop*- Short for the "Tropicana Field," known for its difficult elements that cause players to make elements.  In Jollyballs, this is known as any irregular element in the field of play.
```
Comment (fatcatt316) [inline Glossary.md]: Typo: `cause players to make elements`.

File: Glossary.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+## Offensive Terms
+* *Power Slam* - A power slam is when the back player slams the ball after being set by one of the two front players.  This gives the back player a very strong slam opportunity.
+
+* *Wall Play*- When a player makes a shot off of, or passes the ball off of the wall this is said to be good "wall play."
+
+* *Reverse Dab*- A move used to describe a backhand power slam.
+
+* *Flick-o-da-wrist*- The distinction used to distinguish a normal shot from a slam.
+
+* *Ace*- A return that is unable to be returned.  Most players will have a special dance and occasionally a song that whistled to accompany an ace.
```
Comment (fatcatt316) [inline Glossary.md]: Typo: "a song that whistled".

File: README.md [hunk 8]
```diff
@@ -4,37 +4,64 @@
...
+The first player to call for a match of Jollyballs shall be the first captain. The player who seconds the notion shall be the second captain. Selecting players will then occur in the form of a Snake draft. The first captain drafts a single player, the second drafts two players, and then the remaining undrafted player joins the first captain's team.
 
 ## Choosing which team serves first:
 Before each game in the series, each team places their paddles on the table with their paddle handle hanging off the edge. Two paddles should be placed on the corner and the third paddle should be centered on the table.
 
 One team will toss the jollyball at the other team's paddles. If the first player hits the padding of the paddle, then the other team has a chance for a rebuttal. In the event that the player of the second team hits the paddle or handle, the second team will be first to serve. If the first player hits the handle of any of the second team's paddles, or hits two paddles in one shot, then it will be their ball first.
 
+In the event that every player on each team has had a chance to shoot, the rebuttal is suspended and the first team to hit a paddle wins.
```
Comment (fatcatt316) [inline README.md]: 👏

File: README.md [hunk 9]
```diff
@@ -4,37 +4,64 @@
...
 
-In the event of a slam, the ball can be blocked or deflected back to the side that slammed without the need for a pass. A block does not count against the three maximum touches, nor does it count against the team that slammed. However, if a block is hit back to the team who slammed the ball, that team has to return the ball, as the ball is live.
+If a player is hit while outside the field of play, the ball is dead unless it hits the following locations; the edge of the table, a player's paddle, a player's wrist or hand, or any other body part over the table.
+
+##Blocks
+Any shot, slam, poke, or hit can be blocked, with the exception of a serve. A shot can be blocked or deflected back to the side that hit the ball without the need for a pass and will count as a live ball. A block does not count against the three maximum touches and a player may allow immediately hit the ball after blocking. nor does it count against the team that hit it.
```
Comment (fatcatt316) [inline README.md]: I think you can get rid of that whole last sentence: `. nor does it count against the team that hit it.`

File: README.md [hunk 10]
```diff
@@ -4,37 +4,64 @@
...
+
+*What is a Block?*
+A block is defined as the defensive player extending his/her paddle and holding it out on his/her side of the net with hopes to deflect the shot.  A block may move horizontally along the net and vertically above the net, but not towards the net, or else it is not considered a block and results in a fault.
 
-*Note: While we observe the Supreme Court's opinion of "You know a slam when you see one," a good rule of thumb is "a flick of the wrist" is a slam.*
+If the paddle of the blocker is hit while attempting to block a shot, it counts as his/her deflection and will count against the blocker if it goes off the table.  If any part of the blocker is over the table and hit while blocking, the ball is live and the responsibility the individual whom was hit.
```
Comment (fatcatt316) [inline README.md]: I think you need an "of" in here: `the responsibility the individual`

File: README.md [hunk 11]
```diff
@@ -44,5 +71,6 @@ in possession of the ball and rotates positions clockwise.
 
 *Note: This is meant to be fun, don't be a douche. When in doubt, redo the play, as the ball does not lie.*
 
-## End of Game:
-At the conclusion of the game, players of both sides line up and shake hands. This is an expected and time-honored tradition.
+* If a player hits three consecutive lets, then on the third let within a single possession will count as live.
```
Comment (fatcatt316) [inline README.md]: I think this sentence is wonky. Maybe that last chunk could be `then the third let will count as live.`

File: README.md [hunk 12]
```diff
@@ -44,5 +71,6 @@ in possession of the ball and rotates positions clockwise.
...
+
+* To call time out, all players on the team requesting a time out must place their paddles on the table to signal that a time out is being called.  It is also customary that players kneel during a time out
```
Comment (fatcatt316) [inline README.md]: I'd throw a `.` onto the end o' this (or perhaps even a `!`).

File: Glossary.md [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+
+## Defensive Terms
```
Comment (dneanover) [inline Glossary.md]: Always room to add it!

File: README.md (hunk 9 above)
Comment (dneanover) [inline README.md]: Whoopsie

File: README.md (hunk 4 above)
Comment (dneanover) [inline README.md]: Good point.  Tightening up the language.

**Pull request merged**